lib/glific/clients/common_webhook.ex

```diff
@@ -201,11 +203,17 @@ defmodule Glific.Clients.CommonWebhook do
            Bhasini.with_config_request(
              fields,
              contact.language.locale
-           ) do
-      params = Jason.decode!(response.body)
+           ),
+         %{"feedbackUrl" => _feedback_url, "pipelineInferenceAPIEndPoint" => _endpoint} = params <-
```

Comment:
Do we need the pattern matching if we are using bang version of Jason.decode, since it will anyway raise exception.